lib/embed/viewer/was_seed.rb

```diff
@@ -19,6 +33,13 @@ def self.supported_types
         [:"webarchive-seed"]
       end
       
+      def thumbs_list
+        Embed::WasSeedThumbs.new(@purl_object.druid).get_thumbs_list
+      end
+      
+      def format_memento_datetime(memento_datetime)
+        DateTime.strptime(memento_datetime,'%Y%m%e%H%M%S').strftime('%b %e, %Y')
```

Comment:
Could we use this standardized date/time format?

https://github.com/sul-dlss/sul-embed/blob/df83fbe66d6745462ffbd91232d3ef0193b818e9/lib/embed/viewer/common_viewer.rb#L205-L207